cdap-api/src/main/java/co/cask/cdap/api/schedule/Schedule.java

```diff
@@ -66,38 +68,26 @@ public String getCronEntry() {
   }
 
   @Override
-  public boolean equals(Object o) {
-    if (this == o) {
+  public int hashCode() {
+    return Objects.hashCode(name, description, cronEntry);
```

Comment:
IntelliJ can generate code that can deal with null. Can you not copy the `Objects` class?